packages/main/src/ComboBox.ts

```diff
@@ -1118,15 +1118,16 @@ class ComboBox extends UI5Element implements IFormInputElement {
 
 		const sameItemSelected = this.value === this._selectedItemText;
 		const sameSelectionPerformed = this.value.toLowerCase() === this.filterValue.toLowerCase();
+		const shouldFireSelectionEvent = this.noTypeahead && this._lastValue !== this.value;
```

Comment:
I am not sure if this check is correct. There is a case where we select an item from the picker and then delete last character from the value and select the same item again. Since the typeahead is off, no selection has been performed during the value change therefore I believe the event should be fired just once, but now it's fired twice.